src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -1411,50 +1415,51 @@ private enum GetBytesMode { AllocateArray, Count, Span }
         }
 
         /// <summary>
-        /// Return the value of this BigInteger as a little-endian twos-complement
-        /// uint array, using the fewest number of uints possible. If the value is zero,
-        /// return an array of one uint whose element is 0.
+        /// Converts the value of this BigInteger to a little-endian twos-complement
+        /// uint span allocated by the caller, and shrinks it to the fewest number of uints possible.
         /// </summary>
-        /// <returns></returns>
-        private uint[] ToUInt32Array()
+        /// <param name="buffer">Pre-allocated buffer by the caller.</param>
+        private void CopyTo(ref Span<uint> buffer)
```

Comment:
`ref` is for mutation here because I need to adjust bounds of the original buffer allocated by the caller. This behavior is mentioned in the comment for this method.

Reply:
Return `written` (or count or how you will call it), so the caller can slice then span. See comment above.